earth2studio/data/weatherbench.py

```diff
@@ -0,0 +1,241 @@
+        self.zarr_group = zarr.open(gcstore, mode="r")
+
+    def __call__(
+        self,
+        time: datetime | list[datetime] | TimeArray,
+        variable: str | list[str] | VariableArray,
+    ) -> xr.DataArray:
+        """Function to get data.
+
+        Parameters
+        ----------
+        time : datetime | list[datetime] | TimeArray
+            Timestamps to return data for (UTC).
+        variable : str | list[str] | VariableArray
+            String, list of strings or array of strings that refer to variables to
+            return. Must be in the ARCO lexicon.
+
+        Returns
+        -------
+        xr.DataArray
+            climatology data from WeatherBench climatology
+        """
+        time, variable = prep_data_inputs(time, variable)
+        # Create cache dir if doesnt exist
+        pathlib.Path(self.cache).mkdir(parents=True, exist_ok=True)
+
+        # Fetch index file for requested time
+        data_arrays = []
+        for t0 in time:
+            data_array = self.fetch_wb_dataarray(t0, variable)
+            data_arrays.append(data_array)
+
+        # Delete cache if needed
+        if not self._cache:
+            shutil.rmtree(self.cache)
+
+        return xr.concat(data_arrays, dim="time")
+
+    def fetch_wb_dataarray(
+        self,
+        time: datetime,
+        variables: list[str],
+    ) -> xr.DataArray:
+        """Retrives WeatherBench data array for given date time by downloading a
+        lat lon array from the Zarr store
+
+        Parameters
+        ----------
+        time : datetime
+            Date time to fetch
+        variables : list[str]
+            list of atmosphric variables to fetch.
+
+        Returns
+        -------
+        xr.DataArray
+            WeatherBench data array for given date time
+        """
+        LAT = self.zarr_group["latitude"][:]
+        LON = self.zarr_group["longitude"][:]
+        wbda = xr.DataArray(
+            data=np.empty((1, len(variables), len(LAT), len(LON))),
+            dims=["time", "variable", "lat", "lon"],
+            coords={
+                "time": [time],
+                "variable": variables,
+                "lat": LAT,
+                "lon": LON,
+            },
+        )
+
+        # Load levels coordinate system from Zarr store and check
+        level_coords = self.zarr_group["level"][:]
+        # Get time index (vanilla zarr doesnt support date indices)
+        hour, day_of_year = self._get_time_index(time)
+
+        # TODO: Add MP here
+        for i, variable in enumerate(
+            tqdm(
+                variables,
+                desc=f"Fetching WeatherBench Climatology for {time}",
+                disable=(not self._verbose),
+            )
+        ):
+            logger.debug(
+                f"Fetching WeatherBench Climatology zarr array for variable: {variable} at {time.isoformat()}"
+            )
+
+            try:
+                arco_name, modifier = WBLexicon[variable]  # type: ignore
```

Comment:
Yeah, looking at the pipeline just remove the comment the recommit.

I think its an issue because the lexicon object isnt in the git history yet and it has trouble figuring out the return type. Then when you commit it, its able to access it.